frontend/src/store/userStore.ts

```diff
@@ -1,38 +1,40 @@
 import { ref } from 'vue';
 import { defineStore } from 'pinia';
+import type { User } from '@/interfaces';
 import { userService } from '@/services';
 
 export const useUserStore = defineStore('user', () => {
+  const userSearch = ref([] as User[]);
   const idps = ref([] as Object[]);
   const loading = ref(false);
 
-  async function listIdps() {
+  async function searchUsers(params: Object) {
     try {
       loading.value = true;
-      const res = await userService.listIdps();
-      idps.value = res.data;
+      const userResponse = await userService.searchForUsers(params);
+      userSearch.value = userResponse.data;
     } catch (error) {
-      console.error(`listIdps error: ${error}`);
+      console.error(`searchUsers error: ${error}`);
       // So that a caller can action it
       throw error;
     } finally {
       loading.value = false;
     }
   }
 
-  // Will 404. TODO DELETE
-  async function testBad() {
+  async function listIdps() {
     try {
       loading.value = true;
-      await userService.testBad();
+      const res = await userService.listIdps();
+      idps.value = res.data;
     } catch (error) {
-      console.error(`bad error: ${error}`);
+      console.error(`listIdps error: ${error}`);
```

Comment:
We should look into some kind of frontend console logging library so that we aren't directly using console.* commands (this will be picked up by our linter soon)